projects/ui/src/lib/components/po-accordion/services/po-accordion.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+
+import { Subject } from 'rxjs';
+
+/**
+ * @docsPrivate
+ *
+ * @description
+ *
+ * Serviço para poder notificar o componente `po-accordion` de que um `po-accordion-item` foi
+ * expandido/colapsado.
```

Comment:
O que acham de alterarmos para:
`Serviço para poder notificar o componente `po-accordion` quando um `po-accordion-item` for expandido ou colapsado.`